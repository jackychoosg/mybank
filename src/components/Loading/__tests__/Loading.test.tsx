import { render, cleanup } from '@testing-library/react'
import { Loading } from '../index'
describe('Loading component', () => {
  afterEach(cleanup)

  it('should take a snapshot', () => {
    const { asFragment } = render(<Loading />)

    expect(asFragment()).toMatchSnapshot()
  })
})
