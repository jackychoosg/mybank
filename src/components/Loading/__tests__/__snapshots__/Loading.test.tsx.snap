// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading component should take a snapshot 1`] = `
<DocumentFragment>
  <div
    class="sc-bdnxRM dgqAJz"
  >
    <div
      class="sc-gtsrHT gIWqJW"
    >
      Loading...
    </div>
  </div>
</DocumentFragment>
`;
